src/commonMain/kotlin/scientifik/gdml/solids.kt

```diff
@@ -45,6 +45,79 @@ data class GDMLOrb(
     var r: Number
 ) : GDMLSolid()
 
+@Serializable
+@SerialName("ellipsoid")
+data class GDMLEllipsoid(
+    override var name: String,
+    var ax: Number,
+    var by: Number,
+    var cz: Number,
+    var zcut1: Number = -1000000.0,
```

Comment:
Why such strange number? I think that we should avoid default if it is meaningless and leave `Number?`.

Reply:
i thougth it was default value: ![image](https://user-images.githubusercontent.com/54292574/77736131-e2853f80-701c-11ea-9e74-40c035c087d2.png)

or it`s really meaningless here?

Reply:
I think it is meaningless. They just put a random number to make it optional. @Zelenyy ?

Reply:
It is non trivial question, they can use this value for something.

Reply:
@Zelenyy It seems random. But actually it is not our problem, because we do not need to reproduce GEANT behavior. Setting it to null won't affect serialization and we will have clear indication that it is not defined.

Reply:
so it must be like `var zcut1: Number? = null` or without any initialization with only `Number?` type?

Reply:
I think that Number? = null is a good solution.